### PR TITLE
B 04008 - Design Tweaks

### PR DESCRIPTION
**Pull request by robbiecarlton**
Lots of small visual changes

-  timestamp is located to the right of the message
-  identicon moved to the left of the message (where timestamp currently is)
-  lots of changes to identicon to improve ux and accesibility
-  name has been bolded for chatter's own messages and a bright highlight has been placed around the box containing chatter's own messages
-  chat user names display in the channel message history as they were input by the user, according to whatever case was input
-  last six chars of the DNA hash show in the app information
-  username displays in the top right next to where the profile icon and identicon will be displayed
-  the host ID is shown in the UI tooltip (so we can easily know which host too SSH into if there is an issue)
-  make message boxes smaller
-  Fix + sign issue
-  loading indicators show when waiting for 
    - channels to load
    - channel to be created
    - message to be sent

the holo side requires currently unreleased changes in web-sdk and chaperone. See https://github.com/Holo-Host/web-sdk/pull/34 and https://github.com/Holo-Host/chaperone/pull/133

### CODE REVIEW

File: src/identicon.js [hunk 1]
```diff
@@ -3,14 +3,19 @@ let byteIndex = 0
...
-  const result = bytes[byteIndex] / 256.0
-  byteIndex = (byteIndex + 1) % bytes.length
-  return result
+  const getByte = () => {
+    const result = bytes[byteIndex]
+    byteIndex = (byteIndex + 1) % bytes.length
+    return result
+  }
+
+  return (getByte() + getByte()) / 512 // use 2 bytes per value to make sure we're using all of the hash
```
Comment (timotree3) [inline src/identicon.js:18]: Adding is no good because it means that a hash with bytes `[0, 255, 0, 255 ...]` is equivalent to `[1, 244, 1, 244, ...]`

**Review by timotree3 — CHANGES_REQUESTED** [review]

File: src/identicon.js (hunk 1 above)
Comment (timotree3) [reply src/identicon.js:18]: What you could do instead is
```suggestion
  return (getByte() * 256 + getByte())/(256*256) // use 2 bytes per value to make sure we're using all of the hash
```

Comment (robbiecarlton) [reply src/identicon.js:18]: hmm. I considered that, but then I don't think the least significant byte will actually have any effect on the final image, so it's basically equivalent to just using every other byte. Will think about it more.

Comment (timotree3) [reply src/identicon.js]: XOR is no good either. It has the same many equivalent pairs problem as addition e.g. `251 ^ 4` = `253 ^ 2` = `254 ^ 1` = `255 ^ 0` = `255`

Comment (timotree3) [reply src/identicon.js]: Perhaps it would be possible to take advantage of primes. log<sub>2</sub>(2<sup>A</sup> * 3<sup>B</sup>)

Comment (timotree3) [reply src/identicon.js]: that doesn't seem that good either. maybe (2<sup>A</sup> * 3<sup>B</sup>)/(2<sup>256</sup> * 3<sup>256</sup>)

Comment (robbiecarlton) [reply src/identicon.js]: Decision was to move on for now and revisit when we need it for production.

**Review by timotree3 — APPROVED** [review]

**Comment by timotree3** [comment]
Looks good. Are you going to wait until the web SDK dependency can be bumped to a real version before merging?

**Comment by timotree3** [comment]
Guess not :)

**Pull request merged**